core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -33,6 +33,10 @@ class HttpWorkspaceManagerDAO(baseWorkspaceManagerUrl: String)(implicit val syst
     new ResourceApi(getApiClient(accessToken.token))
   }
 
+  private def getWorkspaceApplicationApi(accessToken: OAuth2BearerToken) = {
```

Comment:
Nit: I'm not convinced that it is worth making a separate method for this, especially given that it's only called in one place.

Reply:
Yeah, I was 51/49 on this one. I did this to be consistent with the rest of the class.